### PR TITLE
Added "Icons in RTL" section to documentation

### PR DESCRIPTION
**Pull request by marcopashkov**

### CODE REVIEW

**Comment by ianegordon** [comment]
LGTM for the iOS section.

File: index.html [hunk 1]
```diff
@@ -32,16 +32,30 @@
...
             <li><a href="#icon-font-for-the-web">Icon fonts for the web</a></li>
             <li><a href="#icon-images-for-the-web">Icon images for the web</a></li>
             <li><a href="#icons-for-android">Icons for Android</a></li>
             <li><a href="#icons-for-ios">Icons for iOS</a></li>
+            <li><a href="#icons-in-rtl">Icons in RTL</a></li>
           </ul>
         </nav>
 
-        <h1 id="what-are-material-icons-">What are material icons?</h1>
+        <style type="text/css">
```
Comment (shyndman) [inline index.html]: It's a little weird for this to be here.

File: index.html [hunk 2]
```diff
@@ -32,16 +32,30 @@
...
+  .fix-table {
+    display: table !important;
+  }
+  .fix-table tr div {
```
Comment (shyndman) [inline index.html]: Is `tr` necessary in this selector?

File: index.html [hunk 3]
```diff
@@ -116,10 +130,10 @@ <h2 id="setup-method-2-self-hosting">Setup Method 2. Self hosting</h2>
   font-weight: 400;
   src: url(https://example.com/MaterialIcons-Regular.eot); /* For IE6-8 */
   src: local(&#39;Material Icons&#39;),
-       local(&#39;MaterialIcons-Regular&#39;),
-       url(https://example.com/MaterialIcons-Regular.woff2) format(&#39;woff2&#39;),
-       url(https://example.com/MaterialIcons-Regular.woff) format(&#39;woff&#39;),
-       url(https://example.com/MaterialIcons-Regular.ttf) format(&#39;truetype&#39;);
+    local(&#39;MaterialIcons-Regular&#39;),
```
Comment (shyndman) [inline index.html]: Why the space removal?

File: www/css/material.css [hunk 4]
```diff
@@ -577,9 +578,7 @@ input[type="submit"] {
 html {
   box-sizing: border-box; }
 
-*,
-*::after,
-*::before {
+*, *::after, *::before {
```
Comment (shyndman) [inline www/css/material.css]: Revert this change block. Newlines after commas, here and below.

File: index.html (hunk 1 above)
Comment (marcopashkov) [inline index.html]: Moved it into the header section. I do not want to move it into the CSS file as it is being generated.

File: index.html (hunk 3 above)
Comment (marcopashkov) [inline index.html]: font_update

File: index.html (hunk 2 above)
Comment (marcopashkov) [inline index.html]: no - removed it.

File: www/css/material.css (hunk 4 above)
Comment (marcopashkov) [inline www/css/material.css]: fixed

File: index.html (hunk 3 above)
Comment (marcopashkov) [inline index.html]: the existing code snippet crops off part of the code sample
